.github/workflows/javascript-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Continuous testing
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use node 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run tests
+        run: yarn test
+      - name: Run lint
+        run: yarn lint
+      - name: Build the project
+        run: yarn build
```

Comment:
Would you consider making these separate jobs? This way tests, linting and building would run in parallel and, most importantly, they will show a bigger picture. 

Consider three scenarios:
* Tests failed, codestyle and build status unknown
* Tests, codestyle, build — all three jobs have failed status
* Tests are red, build and codestyle are green

Two latter cases allow you to draw some conclusions about what's wrong even without reading the output. But they would be hidden behind tests failure in a sequential run.

Reply:
Moved to #8 

Please see if that implementation aligns to what you are referring.